src/settings/Settings.js

```diff
@@ -305,7 +305,7 @@ export const SETTINGS = {
     "VideoView.flipVideoHorizontally": {
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
         displayName: _td('Mirror local video feed'),
-        default: false,
+        default: true,
```

Comment:
This requires a product decision, I think.

Reply:
How do I go about getting one?

Reply:
making noise in the direction of Nad

Reply:
ftr: https://github.com/vector-im/riot-web/issues/10651 and https://github.com/vector-im/riot-web/issues/5633 are relevant issues.

Reply:
@nadonomy for context the current 1:1 video calls have a small local video feed. This feed is not mirrored by default although I think it aught to be. (See the issues linked by turt2live)
Currently it looks like this:
![image](https://user-images.githubusercontent.com/23084468/86810851-3cae8b80-c075-11ea-835c-f05e11d27484.png)

But it's probably a lot more intuitive like this:
![image](https://user-images.githubusercontent.com/23084468/86810907-4801b700-c075-11ea-8900-e87d6c9a4fcf.png)

Should we make the default mirrored?

Reply:
Per the call: we'll keep this off by default for now as the screen sharing issue needs work.

Reply:
Because of https://github.com/vector-im/riot-web/issues/5633 we're not doing this but I'm making it my personal mission to fix that one soon.